src/products/views/ProductList/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { isInEnum } from "@saleor/misc";
+import { ProductTypeKindEnum } from "@saleor/types/globalTypes";
+
+interface ProductKindChoice {
+  label: string;
+  value: ProductTypeKindEnum;
+}
+
+export const getAvailableProductKinds = (): ProductKindChoice[] =>
+  Object.keys(ProductTypeKindEnum).map(kind => ({
+    label: kind,
+    value: kind as ProductTypeKindEnum
+  }));
+
+export const getProductKindOpts = (
+  availableProducts: ProductKindChoice[]
+): ProductKindChoice[] =>
+  availableProducts.map(kind => {
+    switch (kind.value) {
+      case ProductTypeKindEnum.GIFT_CARD:
+        return {
+          ...kind,
+          label: "Gift Card"
```

Comment:
If labels represent constant enum values, which are shown in UI, we should use intl to allow their translation.

Reply:
@orzechdev I have absolutely no idea why GitHub bot didn't notify me about your comment, fixed